openedx/core/djangoapps/course_groups/views.py

```diff
@@ -362,3 +365,61 @@ def debug_cohort_mgmt(request, course_key_string):
         kwargs={'course_key': course_key.to_deprecated_string()}
     )}
     return render_to_response('/course_groups/debug.html', context)
+
+
+@require_http_methods(("GET", "POST"))
+@expect_json
+@login_required
+def cohort_discussion_topics(request, course_key_string):
+    """
+    The restful handler for cohort discussion requests.
+    This will raise 404 if user is not staff or .
+    GET
+        Returns the JSON representation of discussion topics w.r.t categories for the course.
+    POST
+        Updates the cohort_discussion for the course. Returns the JSON representation of updated discussions.
+    """
+    course_key = SlashSeparatedCourseKey.from_deprecated_string(course_key_string)
+    try:
+        course = get_course_by_id(course_key)
+    except Http404:
+        # course didn't exist, or requesting user does not have access to it.
+        return JsonError(status=404)
+
+    discussions_category = get_discussion_category_map(course)
+    discussions_category['coursewide_categories'] = discussions_category['entries']
+    del discussions_category['entries']
```

Comment:
You can do `discussions_category.pop('entries')` instead of the above two lines.